core/domain/exp_services.py

```diff
@@ -1461,17 +1461,18 @@ def accept_suggestion(editor_id, thread_id, exploration_id, commit_message):
         suggestion = feedback_services.get_suggestion(
             exploration_id, thread_id)
         suggestion_author_username = suggestion['author_name']
-        full_commit_message = ('Accepted suggestion by %s: %s' % (
-            suggestion_author_username, commit_message))
         change_list = _create_change_list_from_suggestion(suggestion)
+        suggestion_author_id = user_services.get_user_id_from_username(
+            suggestion_author_username)
         update_exploration(
-            editor_id, exploration_id, change_list, full_commit_message)
+            suggestion_author_id, exploration_id, change_list,
+            commit_message)
         feedback_services.create_message(
-            exploration_id, thread_id, editor_id, None, None,
+            exploration_id, thread_id, editor_id,
+            feedback_models.STATUS_CHOICES_FIXED, None,
             'Suggestion accepted.')
-        thread = feedback_models.FeedbackThreadModel.get_by_exp_and_thread_id(
-            exploration_id, thread_id)
-        thread.status = feedback_models.STATUS_CHOICES_FIXED
+        thread = (feedback_models.FeedbackThreadModel.
```

Comment:
Doesn't look like you need this and the next two lines at all ... you just get the model and then put it, without changing anything.